app_install.sh

```diff
@@ -46,42 +46,48 @@ install_wordpress() {
 # Function to uninstall Nginx on Debian-based systems
 uninstall_nginx_debian() {
     echo "Uninstalling Nginx..."
-    sudo apt-get remove -y nginx
+    #sudo apt-get remove -y nginx
+    /path/to/app_uninstall.sh
```

Comment:
Hey, no need for uninstall function here because it's been moved to other script.